x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/columns.tsx

```diff
@@ -363,6 +381,7 @@ export const getMonitoringColumns = (
         </EuiText>
       ),
       width: '14%',
+      truncateText: true,
```

Comment:
Looks like there's some overlap with the tooltip hover icon:
<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/142949254-2a79a6c2-ee14-45a0-a4e8-ba30e69bc152.png" />
</p>


Looks to be just the `EuiTooltip` ones -- the `Last Gap` custom one seems to be 👍 though:


<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/142949423-462370e1-f337-4e85-a23e-2d7f3db0cea3.png" />
</p>

Reply:
done, fixed that